### PR TITLE
Fix verbosity for virt-controller

### PR DESCRIPTION
**Pull request by xpivarc**
**Special notes for your reviewer**:

**Release note**:
```release-note
None
```

### CODE REVIEW

**Review by ashleyschuett — COMMENTED** [review]
is there a way to unit test this?

**Comment by xpivarc** [comment]
> is there a way to unit test this?

I don't believe so. (Sorry for the late reply)

File: pkg/virt-controller/watch/application.go [hunk 1]
```diff
@@ -325,7 +325,7 @@ func (vca *VirtControllerApp) configModificationCallback() {
 
 // Update virt-controller log verbosity on relevant config changes
 func (vca *VirtControllerApp) shouldChangeLogVerbosity() {
-	verbosity := vca.clusterConfig.GetVirtHandlerVerbosity(vca.host)
+	verbosity := vca.clusterConfig.GetVirtControllerVerbosity(vca.host)
```
Comment (vladikr) [inline pkg/virt-controller/watch/application.go:328]: :facepalm:

**Pull request merged**